### ISSUE TITLE
There seems to be a bug when opening the annotation file and visually adjust the size and location of the previously labeled bboxes？

### ISSUE THREAD

**Issue by George-Zhu** [tzutalin/labelImg#18]
Hi @tzutalin ~ This labeling tool is pretty convenient and I like it very much!  However, when I click the "open annotation" button, open a previously saved xml file and try to adjust the size and location of the bbox visually, the shape of the bboxes are no longer rectangles. Maybe there is a bug here.

I did this because I wanted to use CNN to pre-label the bboxes so that we can just make some adjustment, which can reduce the workload.

I 'm looking forward to your reply : )

**Comment by George-Zhu** [comment]
I have fixed the bug by modifying the Line 121 of pascal_voc_io.py as following:
points = [(xmin,ymin), (xmin,ymax), (xmax, ymax), (xmax, ymin)]   -->
points = [(xmin, ymin), (xmax, ymin), (xmax, ymax), (xmin, ymax)] 

Anyway, thanks a lot :)

**Comment by tzutalin** [comment]
Hi @George-Zhu ,
 Can you share me with the examples which have problems? Or do you think it is ok to use the below changes without side effect?
points = [(xmin,ymin), (xmin,ymax), (xmax, ymax), (xmax, ymin)] -->
points = [(xmin, ymin), (xmax, ymin), (xmax, ymax), (xmin, ymax)]

**Comment by nkhdiscovery** [comment]
@George-Zhu Thanks for the fix! Saved my day!
@tzutalin @George-Zhu : I wonder why is this caused during parsing the xml with wrong order of points, I guessed there was an issue with the type of shape or something, then I found this solution and wondered, can you explain the cause?

**Comment by nkhdiscovery** [comment]
@tzutalin You can just load an image, annotate, save and then close the project. ( I did this with Open Dir and set the default annotation folder). Close it and re-open with Open Dir, then try to edit rectangles ... you'll see that it fails to keep the aspect ratio and messes the box.

**Comment by nkhdiscovery** [comment]
Hi @tzutalin ,
Yes it worked, I'm using it right now. 
Great work by the way :)
I will spend a pull request if I improved anything, but not for this issue right now as I still have not figured the reason well. 

Actually, I think this has no further harm as I had a quick review of the code, changing this line makes the order clock-wise and later in adding these points to shape and building the canvas (which is handled by Qt I guess), the resulting canvas would be obviously treated differently. 
I let you know more if I figured it completely.

**Comment by tzutalin** [comment]
@nkhdiscovery ,
   Very appreciated ! I am looking forward to your PR!!!!

**Comment by nkhdiscovery** [comment]
Hi @tzutalin ,
Yes it worked, I'm using it right now. 
Great work by the way :)
I will spend a pull request if I improved anything, but not for this issue right now as I still have not figured the reason well. 

Actually, I think this has no further harm as I had a quick review of the code, changing this line makes the order clock-wise and later in adding these points to shape and building the canvas (which is handled by Qt I guess), the resulting canvas would be obviously treated differently. 
I let you know more if I figured it completely.
